src/frontend/js/widgets/Dashboard/components/DashboardItem/Order/OrderStateLearnerMessage/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { defineMessages } from 'react-intl';
+import OrderStateMessage, { OrderStateMessageBaseProps } from '../OrderStateMessage';
+
+export const messages = defineMessages({
+  statusDraft: {
+    id: 'components.DashboardItem.Order.OrderStateMessage.statusDraft',
+    description: 'Status shown on the dashboard order item when order is draft.',
+    defaultMessage: 'Draft',
+  },
+  statusSubmitted: {
+    id: 'components.DashboardItem.Order.OrderStateMessage.statusSubmitted',
+    description: 'Status shown on the dashboard order item when order is submitted.',
+    defaultMessage: 'Submitted',
+  },
+  statusPending: {
+    id: 'components.DashboardItem.Order.OrderStateMessage.statusPending',
+    description: 'Status shown on the dashboard order item when order is pending.',
+    defaultMessage: 'Pending',
+  },
+  statusOnGoing: {
+    id: 'components.DashboardItem.Order.OrderStateMessage.statusOnGoing',
+    description:
+      'Status shown on the dashboard order item when order is validated with no certificate',
+    defaultMessage: 'On going',
+  },
+  statusCompleted: {
+    id: 'components.DashboardItem.Order.OrderStateMessage.statusCompleted',
+    description:
+      'Status shown on the dashboard order item when order is validated with certificate',
+    defaultMessage: 'Completed',
+  },
+  statusWaitingSignature: {
+    id: 'components.DashboardItem.Order.OrderStateMessage.statusWaitingSignature',
+    description:
+      "Status shown on the dashboard order item when order is validated with contract's learner signature missing.",
+    defaultMessage: 'Signature required',
+  },
+  statusWaitingCounterSignature: {
+    id: 'components.DashboardItem.Order.OrderStateMessage.statusWaitingCounterSignature',
+    description:
+      "Status shown on the dashboard order item when order is validated with contract's organization signature missing.",
+    defaultMessage: 'On going',
```

Comment:
```suggestion
    defaultMessage: 'Pending for organization's signature',
```

Reply:
it's not a bug, it's a feature.
The learner don't have to wait the organization to sign to access the training.
For it point of view, this state is already an "On Going" state